.gitignore

```diff
@@ -11,4 +11,3 @@ dist/
 docs-built/
 tmp-bower-repo/
 tmp-docs-repo/
-.babel-cache/
```

Comment:
I'm wondering if we should leave this in for a while until this change makes it into the master branch and has been there for a week or two. Mainly because I can see people accidentally adding this in a commit and submitting it in a PR. It won't hurt anything to be there for a time, and we can create an issue that we target with the v0.25 release to remember to remove it.